pkg/controllers/job/state/aborting.go

```diff
@@ -31,17 +33,18 @@ func (ps *abortingState) Execute(action vkv1.Action) error {
 		// Already in Restarting phase, just sync it
 		return KillJob(ps.job, func(status *vkv1.JobStatus) {
 			status.State.Phase = vkv1.Restarting
+			status.State.LastTransitionTime = metav1.Now()
```

Comment:
There a lots of codes:
```
status.State.LastTransitionTime = metav1.Now()
```
which can be upgraded into a function which used to update status and update the `LastTransitionTime`  when required.